### ISSUE TITLE
Incompatible with Android 14

### ISSUE THREAD

**Issue by eylenburg** [espeak-ng/espeak-ng#1862]
F-Droid just gives me the "incompatible" warning, "built for an older version of Android". If forced to install, it ends with a non-descript error message. I think Android 14 (and newer versions) now requires a minimum Android API version for apps. The only workaround is installing via `adb`.

**Comment by licaon-kter** [comment]
Yup, only apps built with **targetSDK 23 or later** _(not related to minimum supported)_ can be installed on Android 14.

**Comment by Gamer750** [comment]
Please support android 14

**Comment by RetroBunn** [comment]
eSpeak does support Android 14.
I’m running it on a Samsung Galaxy S22 Ultra. You may need to compile it from source via Gradle or Android Studio.

**Comment by licaon-kter** [comment]
Then let's rename this to _"Tag a new release"_?

**Comment by Gamer750** [comment]
> eSpeak does support Android 14. I’m running it on a Samsung Galaxy S22 Ultra. You may need to compile it from source via Gradle or Android Studio.

Just a question did you delete one ui in your phone did the multi tasking feature is working when I delete one ui the multi tasking feature was not working

**Comment by RetroBunn** [comment]
That might be out of scope for this discussion.

**Comment by licaon-kter** [comment]
Also note that many newer devices that will get Android 14 (stock or custom) might end up arm64 only so the current espeak can't be installed unless it's built for that arch too.

**Comment by DocSniper** [comment]
As there is no 64-bit version at the moment, I compiled a transitional version 1.52-dev today, which also runs on ARMv9 CPUs (64-bit only).

Everything works fine on my device with ARMv9, if anyone wants, the APK can be downloaded here: [espeak-052-dev.zip](https://github.com/user-attachments/files/16348039/espeak-052-dev.zip)

**Comment by MessieAss** [comment]
It installed fine, but i fail at installing voices. Any help?

**Comment by MessieAss** [comment]
Okay, i somehow figured it out. I'd love to tell you how i did it, but suddenly it worked! System is Android14 on a Fairphone 4 and it finally started talking to me. The only thing which comes to mind is that i temporarily installed the Google TTS and after deinstallation, i re- selected the espeak engine and could suddenly select my language. Thank you so much, DocSniper!

**Comment by cg9999** [comment]
For me, the trick to get the voices showing up was:

eSpeak -> three dot menu -> General TTS settings -> Preferred engine -> tap on eSpeak again (even if it's the only one and already selected)

After this, a small window briefly flashed and now the languages are available.

**Comment by valdisvi** [comment]
You can try https://odo.lv/ftp/tools/espeak-release-signed.apk
